packaging/resource/cassandra-reaper-cassandra-ssl.yaml

```diff
@@ -10,19 +10,10 @@ hangingRepairTimeoutMins: 30
 storageType: cassandra
 enableCrossOrigin: true
 incrementalRepair: false
+allowUnreachableNodes: false
```

Comment:
The `allowUnreachableNodes` property was removed in the most recent commits to master.